tests/t136_dynamic.py

```diff
@@ -24,4 +24,5 @@ def build(self, name, cflags='', ldflags=''):
         return TestBase.build(self, name, cflags, ldflags)
 
     def setup(self):
+        self.p_flag = ''
         self.option = '-P "a.?" --no-libcall'
```

Comment:
I have tried many different ways, but I think this is the most clear way because the test author should disable the unexpected patching when testing dynamic tracing feature. This is only needed in dynamic tracing tests so won't be needed other general tests.

Reply:
Doesn't this work?  If the option already has any of `-P` option, it means dynamic tracing test and then we can disable any p_flags, no?

```python
if '-P' in self.option:
    self.p_flag = ''
```

Reply:
I misunderstood the code location. I've changed it as you mentioned. Thanks.